src/main/java/org/openstreetmap/atlas/geography/atlas/raw/sectioning/WaySectionProcessor.java

```diff
@@ -756,6 +778,23 @@ private boolean locationIsPartOfAnIntersectingEdgeOfTheSameLayer(final Location
                 });
     }
 
+    private void logTaskCompletionAsInfo(final String taskName, final String shardName,
+            final Duration duration)
+    {
+        logger.info(COMPLETED_TASK_MESSAGE, taskName, shardName, duration);
+    }
+
+    private void logTaskCompletionAsTrace(final String taskName, final String shardName,
+            final Duration duration)
+    {
+        logger.trace(COMPLETED_TASK_MESSAGE, taskName, shardName, duration);
+    }
+
+    private void logTaskStartedAsInfo(final String taskname, final String shardName)
```

Comment:
You could have this method log the start and return the timer!

Reply:
Good idea! This is cleaner.